### PR TITLE
[Translatable] method setTranslations added

### PR DESCRIPTION
**Pull request by wolfwolker**
Added setter on translatableMethods trait.
Usefull to set multiple translations at same time. (Example: using nelmio/alice to load fixtures of translatable entities)

### CODE REVIEW

File: src/Model/Translatable/TranslatableMethods.php [hunk 1]
```diff
@@ -56,6 +56,28 @@ public function addTranslation($translation)
...
+     *
+     * @return $this
+     */
+    public function setTranslations($translations)
+    {
+        if (!$translations instanceof \Traversable && !is_array($translations)) {
+            throw new \InvalidArgumentException('$translations parameter must be iterable');
+        }
+
+        $this->translations = new ArrayCollection();
```
Comment (docteurklein) [inline src/Model/Translatable/TranslatableMethods.php]: woujdn't this break persistent collections? Doctrine would get lost.

**Comment by docteurklein** [comment]
Can you add a test case please?

File: src/Model/Translatable/TranslatableMethods.php (hunk 1 above)
Comment (wolfwolker) [inline src/Model/Translatable/TranslatableMethods.php]: I don't get you. Why?

Comment (docteurklein) [inline src/Model/Translatable/TranslatableMethods.php]: I'm not sure doctrine will like to replace a persistent collection with another collection. That's why I'd like to see an integration test.

Comment (Einenlum) [reply src/Model/Translatable/TranslatableMethods.php]: Indeed, I think you need to empty the current collection and replace its content.

**Comment by TomasVotruba** [comment]
Replaced by https://github.com/KnpLabs/DoctrineBehaviors/pull/460, that is rebased on current `master` where so much has changed since.

Thank you for your contribution :+1:

**Pull request closed without merging**